src/core/resolvers.ts

```diff
@@ -48,8 +48,13 @@ export function resolversAddon(resolvers: Resolver[]): Addon {
       const dynamic: ImportExtended[] = []
       const sideEffects: ImportExtended[] = []
       await Promise.all([...names].map(async (name) => {
-        if (matched.find(i => i.as === name))
+        const matchedImport = matched.find(i => i.as === name)
+        if (matchedImport) {
+          if (Object.hasOwn(matchedImport, 'sideEffects'))
```

Comment:
```suggestion
          if ('sideEffects' in matchedImport)
```

`Object.hasOwn` is only available in v16, https://node.green/#ES2022-features-Object-hasOwn